### DISCUSSION TITLE
NFT plots created on harvesters not being counted in Total Plot Count nor Pool Count (randomly)

### DISCUSSION THREAD

**Discussion by ostunilima** [Chia-Network/chia-blockchain#7312] [Support]
After installing 1.2 the total number of plots on my farm (1 main node + 2 harvesters) was initially accounted for correctly. Randomly the the total number of plots changes to not account for the NFT plots created on harvesters with MadMAx. Sometimes if I hit the Keys icon, it seems to refresh and show correct count on main GUI

![Capture1](https://user-images.githubusercontent.com/77932678/125130971-f6e53280-e0cf-11eb-88b5-214139c32782.PNG)
. Total plots are still passing filter, but it seems that when the count changes, it does affect Space Pool total estimated plot size as well.

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
